src/ring_theory/graded_algebra/homogeneous_localization.lean

```diff
@@ -226,6 +228,10 @@ numerator and denominator are of the same grading.
 def val (y : homogeneous_localization 𝒜 x) : at x :=
 quotient.lift_on' y (num_denom_same_deg.embedding 𝒜 x) $ λ _ _, id
 
+@[simp] lemma val_mk' (i : num_denom_same_deg 𝒜 x) :
```

Comment:
Is the RHS really simpler than the LHS?

Reply:
This lemma looks good to me, after doing a quotient induction you want lemmas to unfold anything defined in terms of `quotient.lift`